server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -50,6 +68,11 @@ case class NettyConfig(
 
   def socketBacklog(s: Int): NettyConfig = copy(socketBacklog = s)
 
+  def withRequestTimeout(r: Int): NettyConfig = copy(requestTimeout = Some(r.seconds))
+  def withConnectionTimeout(c: Int): NettyConfig = copy(connectionTimeout = Some(c.seconds))
+  def withSocketTimeout(s: Int): NettyConfig = copy(socketTimeout = Some(s.seconds))
+  def withLingerTimeout(l: Int): NettyConfig = copy(requestTimeout = Some(l.seconds))
```

Comment:
You should also have methods like `withNoConnectionTimeout` that just set value to None.

Reply:
@katlasik it's `None` by default and it means "use whatever netty has by default underneath". A method called `withNoConnectionTimeout` suggests that someone might want no timeout at all. Like infinite timeout. Is that what you're proposing here?

Reply:
no, you're right, there are only such methods for booleans in other configs